### ISSUE TITLE
Compatibility with BiG-SCAPE

### ISSUE THREAD

**Issue by dfl5440** [chevrettelab/gator-gc#1]
Hi, I saw your poster from a recent conference and I am interest in using the tool for my own data analysis however I was wondering if the tool produced .gbk files that would be compatible with BiG-SCAPE?

**Comment by chevrm** [comment]
Yup. All gbks for identified windows are available by default. These should be usable in other downstream processing like BIG-SCAPE. However, you may get what you're looking for just by default with the Gator Focal Score calculation, which gives a score for relatedness between windows. This is the number that underlies the heatmap, and you could easily view this as a network if preferred.
